modules/statics/src/account.ts

```diff
@@ -183,6 +187,28 @@ export class HederaCoin extends AccountCoinToken {
   }
 }
 
+/**
+ * The Algo network supports tokens (assets)
+ * Algo tokens work similar to native ALGO coin, but the token name is determined by the chain,
+ * and the unique asset id on the chain: (t)algo:<assetId>
+ */
+export class AlgoCoin extends AccountCoinToken {
+  public assetURL: string;
+
+  constructor(options: AlgoCoinConstructorOptions) {
+    super({
+      ...options,
+    });
+
+    const domainPattern = /^(?:[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?\.)+[a-z0-9][a-z0-9-]{0,61}[a-z0-9]$/;
```

Comment:
could we add a comment for a domain that would match this regex? Or the source if you pulled it from somewhere.

Reply:
I have updated the implementation to use a URL class instead of regex, which was borrowed from steller